pkg/combustion/templates/15-rke2-multi-node-installer.sh.tpl

```diff
@@ -23,6 +23,11 @@ mount /var
 mkdir -p /var/lib/rancher/rke2/agent/images/
 cp {{ .imagesPath }}/* /var/lib/rancher/rke2/agent/images/
 
+{{ if .manifestsPath }}
+mkdir -p /var/lib/rancher/rke2/server/manifests/
```

Comment:
Did we confirm if this is something that we want to perform on all nodes as opposed to just agents for example?